### PR TITLE
Add more docs

### PR DESCRIPTION
**Pull request by torkleyy**

### CODE REVIEW

File: src/lib.rs [hunk 1]
```diff
@@ -1,3 +1,60 @@
+//! RON is a simple config format which looks similar to Rust syntax.
```
Comment (kvark) [inline src/lib.rs]: I think having a block comment aka `/*! ... !*/` would be more convenient for this block

**Comment by kvark** [comment]
Niice
bors r+

**Comment by torkleyy** [comment]
bors is stuck..

@notriddle I cannot find anything about this PR in bors' history..is there a more detailed log?

**Comment by notriddle** [comment]
That history page clearly needs timestamps. Luckily, the database should have enough info to do that (Ecto puts in updated/created columns by default). https://github.com/bors-ng/bors-ng/issues/257

I'm guessing that the crash line just before #33 is the cause of this. When it crashes like that, the queue gets thrown out; this is specifically so that bors doesn't actually get "stuck."

**Comment by torkleyy** [comment]
I see, thanks!

**Pull request merged**